skema/img2mml/eqn2mml.py

```diff
@@ -5,73 +5,101 @@
-              "description": "Lotka-Volterra (eq2)",
-              "value": "\\frac{\\delta y}{\\delta t} = {\\alpha x y} - {\\gamma y}"
-          },
-          "simple": {
-              "summary": "A familiar equation",
-              "description": "A simple equation (mass-energy equivalence)",
-              "value": "E = mc^{2}",
-          },
-          "complex": {
-              "summary": "A more feature-rich equation (Bayes' rule)",
-              "description": "A equation drawing on latex features",
-              "value": "\\frac{P(\\textrm{a } | \\textrm{ b}) \\times P(\\textrm{b})}{P(\\textrm{a})}",
-          }
-      },
-  ),
+    Text,
+    Query(
+        examples={
+            "lotka eq1": {
+                "summary": "Lotka-Volterra (eq1)",
+                "description": "Lotka-Volterra (eq1)",
+                "value": "\\frac{\\delta x}{\\delta t} = {\\alpha x} - {\\beta x y}",
+            },
+            "lotka eq2": {
+                "summary": "Lotka-Volterra (eq2)",
+                "description": "Lotka-Volterra (eq2)",
+                "value": "\\frac{\\delta y}{\\delta t} = {\\alpha x y} - {\\gamma y}",
+            },
+            "simple": {
+                "summary": "A familiar equation",
+                "description": "A simple equation (mass-energy equivalence)",
+                "value": "E = mc^{2}",
+            },
+            "complex": {
+                "summary": "A more feature-rich equation (Bayes' rule)",
+                "description": "A equation drawing on latex features",
+                "value": "\\frac{P(\\textrm{a } | \\textrm{ b}) \\times P(\\textrm{b})}{P(\\textrm{a})}",
+            },
+        },
+    ),
 ]
 
 
-
 def process_latex_equation(eqn: Text) -> Response:
     """Helper function used by both GET and POST LaTeX equation processing endpoints"""
     res = get_mathml_from_latex(eqn)
     return Response(content=res, media_type="application/xml")
 
-@router.get("/img2mml/healthcheck", summary="Check health of eqn2mml service", response_model=int, status_code=200)
+
+@router.get(
+    "/img2mml/healthcheck",
+    summary="Check health of eqn2mml service",
+    response_model=int,
+    status_code=200,
+)
 def img2mml_healthcheck() -> int:
     return 200
 
-@router.get("/latex2mml/healthcheck", summary="Check health of mathjax service", response_model=int, status_code=200)
+
+@router.get(
+    "/latex2mml/healthcheck",
+    summary="Check health of mathjax service",
+    response_model=int,
+    status_code=200,
+)
 def latex2mml_healthcheck() -> int:
     try:
-      return int(requests.get(f"{SKEMA_MATHJAX_ADDRESS}/healthcheck").status_code)
+        return int(requests.get(f"{SKEMA_MATHJAX_ADDRESS}/healthcheck").status_code)
     except:
-      return 500
+        return 500
+
 
 @router.post("/image/mml", summary="Get MathML representation of an equation image")
-async def post_image_to_mathml(data: schema.ImageBytes) -> Response:
```

Comment:
Strange that this stopped working...

Reply:
Yeah, I don't know why schema.ImageBytes didn't work anymore. But, I changed the type of data to UploadFile. And then, all tests worked fine for me.